scripts/ci/pre_commit/pre_commit_check_provider_yaml_files.py

```diff
@@ -38,6 +39,9 @@
     from airflow_breeze.utils.docker_command_utils import get_extra_docker_flags
     from airflow_breeze.utils.run_utils import get_ci_image_for_pre_commits, run_command
 
+    cmd = "python3 /opt/airflow/scripts/in_container/run_provider_yaml_files_check.py"
+    if len(sys.argv) > 1:
+        cmd = cmd + " " + " ".join([shlex.quote(f) for f in sys.argv[1:]])
```

Comment:
Better to use shlex.join

Reply:
Right 🤦

Reply:
Ah no... shlex.join is only available in 3.8 (thank yoy mypy for reminding me).